src/persist-client/src/cache.rs

```diff
@@ -116,10 +120,43 @@ impl PersistClientCache {
                 Arc::clone(x.insert(blob))
             }
         };
+        // WIP put this before or after we wrap it in MetricsBlob?
+        self.rtt_latency_tasks.push(Arc::new(
+            blob_rtt_latency_task(Arc::clone(&blob), Arc::clone(&self.metrics)).await,
+        ));
         Ok(Arc::new(MetricsBlob::new(blob, Arc::clone(&self.metrics))))
     }
 }
 
+async fn blob_rtt_latency_task(
+    blob: Arc<dyn Blob + Send + Sync>,
+    metrics: Arc<Metrics>,
+) -> JoinHandle<()> {
+    mz_ore::task::spawn(|| "persist::blob_rtt_latency", async move {
+        const MEASUREMENT_INTERVAL: Duration = Duration::from_secs(10);
```

Comment:
if this is a gauge I don't think there's any value in polling more frequently than our Prometheus scrape interval